examples/dynamic-client/cluster_scoped_custom_resource.py

```diff
@@ -0,0 +1,152 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This example demonstrates the following:
+    - Creation of a cluster scoped custom resource definition (CRD) using dynamic-client
+    - Creation of custom resources (CR) using the above created CRD
+    - List, patch(update), delete the custom resources
+    - Delete the custom resource defintion
+"""
+
+from kubernetes import config, dynamic
+from kubernetes.dynamic.exceptions import ResourceNotFoundError
+from kubernetes.client import api_client
+import time
+
+
+def main():
+    # Creating a dynamic client
+    client = dynamic.DynamicClient(
+        api_client.ApiClient(configuration=config.load_kube_config())
+    )
+
+    # fetching the custom resource definition (CRD) api
+    crd_api = client.resources.get(
+        api_version="apiextensions.k8s.io/v1beta1", kind="CustomResourceDefinition"
+    )
+
+    # Creating a Namespaced CRD named "ingressroutes.apps.example.com"
+    name = "ingressroutes.apps.example.com"
+
+    crd_manifest = {
+        "apiVersion": "apiextensions.k8s.io/v1beta1",
+        "kind": "CustomResourceDefinition",
+        "metadata": {
+            "name": name,
+        },
+        "spec": {
+            "group": "apps.example.com",
+            "names": {
+                "kind": "IngressRoute",
+                "listKind": "IngressRouteList",
+                "plural": "ingressroutes",
+                "singular": "ingressroute",
+            },
+            "scope": "Cluster",
+            "version": "v1",
+            "subresources": {"status": {}},
+        },
+    }
+
+    crd_creation_respone = crd_api.create(crd_manifest)
+    print(
+        "\n[INFO] custom resource definition `ingressroutes.apps.example.com` created\n"
+    )
+    print("%s\t\t%s" % ("SCOPE", "NAME"))
+    print(
+        "%s\t\t%s\n"
+        % (crd_creation_respone.spec.scope, crd_creation_respone.metadata.name)
+    )
+
+    # Fetching the "ingressroutes" CRD api
+
+    try:
+        ingressroute_api = client.resources.get(
+            api_version="apps.example.com/v1", kind="IngressRoute"
+        )
+    except ResourceNotFoundError:
+        # Need to wait a sec for the discovery layer to get updated
+        time.sleep(2)
+
+    ingressroute_api = client.resources.get(
+        api_version="apps.example.com/v1", kind="IngressRoute"
+    )
+
+    # Creating a custom resource (CR) `ingress-route-*`, using the above CRD `ingressroutes.apps.example.com`
+
+    ingressroute_manifest_one = {
+        "apiVersion": "apps.example.com/v1",
+        "kind": "IngressRoute",
+        "metadata": {
+            "name": "ingress-route-one",
+        },
+        "spec": {},
+    }
+
+    ingressroute_manifest_second = {
```

Comment:
use ingressroute_manifest_two to be consistent with ingressroute_manifest_one above, or use ingressroute_manifest_first above

Reply:
Thank you pointing. I changed the `ingressroute_manifest_one` to `ingressroute_manifest_first`. Have also updated the output screenshots in the description.